### PR TITLE
chore(vendor): bump kind to latest

### LINKED ISSUE #60: Integration: Load kind images using the kind load command
We use kind for integration tests against a kubernetes cluster. We now have a dirty hack to load the built images into the kind docker daemon since when we started using it kind was at a very early stage and didn't have the `load` command. Now the load command is available so we can just replace the hack with it!

Docs here
https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster

### PR DESCRIPTION
**Pull request by leogr**
Attempt to update kind to v0.5.1 and partially fix for #60 (refs #60)
Signed-off-by: Leonardo Grasso <me@leonardograsso.com>

### CODE REVIEW

**Comment by dalehamel** [comment]
Thanks for this, I'll run the integration tests locally to make sure it's all good, but CI seems happy so I'm not too concerned.

File: integration/suite_test.go [hunk 1]
```diff
@@ -34,28 +36,35 @@ func init() {
...
 
 	c.Assert(err, check.IsNil)
 
+	// copy the bpftrace into a tar
+	dir, err := fs.TempDir("", "image-tar")
+	c.Assert(err, check.IsNil)
+	defer os.RemoveAll(dir)
+	imageTarPath := filepath.Join(dir, "image.tar")
+
+	fmt.Println(cmd.ImageNameTag)
```
Comment (leodido) [inline integration/suite_test.go]: Is this a left-over print?

Comment (leogr) [reply integration/suite_test.go]: Oh sorry, fixed!

**Comment by leodido** [comment]
@leogr this needs a final rebase only!

Thanks!

**Comment by leogr** [comment]
Done :)

**Pull request merged**